src/libraries/System.IO.Pipes/src/System/IO/Pipes/NamedPipeClientStream.Windows.cs

```diff
@@ -110,7 +110,7 @@ private bool TryConnect(int timeout)
                 _pipeFlags |= (((int)_impersonationLevel - 1) << 16);
             }
 
-            SafePipeHandle handle = CreateNamedPipeClient(_normalizedPipePath, ref secAttrs, _pipeFlags, _accessRights);
```

Comment:
When the logic is not around, it's better using assert instead of the suppression operator. One assert can cover all subsequent usages.